.vscode/tasks.json

```diff
@@ -1,137 +1,122 @@
 {
-  "version": "2.0.0",
-  "tasks": [
-      {
-          "label": "CompileTextMate",
-          "type": "npm",
-          "script": "compile:TextMate",
-          "problemMatcher": ["$eslint-stylish"],
-          "options": {
-              "cwd": "${workspaceFolder}/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/"
-          }
-      },
-      {
-          "label": "CompilePackage",
-          "command": "dotnet",
-          "args": [
-              "build"
-          ],
-          "options": {
-              "cwd": "${workspaceFolder}/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/"
-          },
-          "problemMatcher": "$tsc-watch",
-          "presentation": {
-              "reveal": "silent"
-          }
-      },
-      {
-          "label": "CompileLibrary",
-          "command": "dotnet",
-          "args": [
-              "build"
-          ],
-          "options": {
-              "cwd": "${workspaceFolder}/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/"
-          },
-          "problemMatcher": "$tsc-watch",
-          "presentation": {
-              "reveal": "silent"
-          },
-          "dependsOn": [
-              "ActivateDotnet"
```

Comment:
Is there a reason this was removed?

For context: I included this when doing the remote development work. When you open the aspnetcore-tooling repo in a devcontainer, you lose the activated dotnet environment that existed in your local session so all builds failed. Having this here ensured that the activated dotnet was used in that scenario.

Reply:
oh interesting, I couldn't run the extension m ore then once. Activate hangs on second execution

Reply:
Hmmm -- are you referring to the dev container extension? In the absence of this, we can always document this quirk with needing to re-activate the environment after launching in the container to make sure folks are aware.

Reply:
Made it so activate only runs on Linux!